robotools/tests.py

```diff
@@ -247,66 +247,79 @@ def test_context(self):
-        evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume='15')
-        evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=20)
-        evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=23.78)
-        evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=numpy.array(23.4))
+            evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume='bla')
+        evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume='15')
+        evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=20)
+        evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=23.78)
+        evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=numpy.array(23.4))
         
         with self.assertRaises(ValueError):
-            evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, liquid_class=None)
+            evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, liquid_class=None)
         with self.assertRaises(ValueError):
-            evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, liquid_class='liquid;class')
-        evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, liquid_class='valid liquid class')
+            evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, liquid_class='liquid;class')
+        evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, liquid_class='valid liquid class')
         with self.assertRaises(ValueError):
-            evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, tip=None)
+            evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, tip=None)
         with self.assertRaises(ValueError):
-            evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, tip=12)
-        evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, tip=4)
-        evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, tip=evotools.Tip.T5)
+            evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, tip=12)
         with self.assertRaises(ValueError):
-            evotools._prepate_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, rack_id=None)
+            evotools._prepare_aspirate_dispense_parameters(rack_label='WaterTrough', position=1, volume=15, tip=[1, 2.6])
```

Comment:
Please match the exception message. You can use pytest: `pytest.raises(ValueError, match="some part of the message")`